pages/linux/runuser.md

```diff
@@ -0,0 +1,24 @@
+# runuser
+
+> Run commands as a specific user and group IDs.
```

Comment:
Perhaps this will read better?
```suggestion
> Run commands as specific users and group IDs.
```

Reply:
`runuser` only run as a specific user at one time, not users, disagree.